bmi_sdk/tests/test_bmi_time.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from pydantic import ValidationError
+
+from ..bmi_time import BmiTime
+
+
+@pytest.mark.parametrize(
+    "unit",
+    [
+        "s",
+        "sec",
+        "second",
+        "seconds",
+        "min",
+        "minute",
+        "minutes" "h",
+        "hr",
+        "hour",
+        "hours",
+        "d",
+        "day",
+        "days",
+    ],
```

Comment:
```suggestion
    "unit",
    [
        "s",
        "sec",
        "second",
        "seconds",
        "min",
        "minute",
        "minutes",
        "h",
        "hr",
        "hour",
        "hours",
        "d",
        "day",
        "days",
    ],
```

Reply:
Hmmm, yeah it looks like there was a missing `,` in the original code before I formatted it. Great catch!